Towny/src/main/java/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -1212,7 +1225,12 @@ public void parseAdminTownCommand(CommandSender sender, String[] split) throws T
 			break;
 		case "set":
 			checkPermOrThrow(sender, PermissionNodes.TOWNY_COMMAND_TOWNYADMIN_TOWN_SET.getNode());
-			TownCommand.townSet(sender, StringMgmt.remArgs(split, 2), true, town);
+			if (split.length == 2) {
+				// Empty Case: /townyadmin town [town] set
+				TownCommand.townSet(sender, StringMgmt.remArgs(split, 2), true, town);
+			} else {

```

Comment:
This should get handled in the parseAdminTownSet method, with a new HelpMenu made for the `/ta town NAME set` command.